.devcontainer/base.Dockerfile

```diff
@@ -42,3 +42,4 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && if [ "${VERSION_CODENAME}" = "bullseye" ]; then apt-get -y install --no-install-recommends python-is-python3; fi \
     # Clean up
     && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/* /root/.gnupg /tmp/library-scripts
+RUN npm install -g vite
```

Comment:
Define which version to install

Reply:
I think no need to install vite. Then I would like to remove this line out